addons/mail/static/src/models/composer/composer.js

```diff
@@ -599,6 +709,25 @@ function factory(dependencies) {
             return [['unlink', unmentionedChannels]];
         }
 
+        /**
+         * Convert text message to HTML.
+         * @returns {String}
```

Comment:
- empty newline between comment and tags
- missing `@private` tag
- Would name it better so that we know it's not destructive action. And message !== composer text content. Avoid using `body` and `message` when it doesn't really fit here.

Reply:
Please, provide the name you think fit better, because I have no idea.

Reply:
what about `getPrettyTextInputContent`?